<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(255,250,220);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="styleSheet">
          <string notr="true">
    font: bold 14px;</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="texts">
          <attribute name="title">
           <string>Texts</string>
          </attribute>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>0</y>
             <width>391</width>
             <height>561</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="0">
             <widget class="QTextBrowser" name="clipText_3"/>
            </item>
            <item row="1" column="0">
             <widget class="QTextBrowser" name="clipText"/>
            </item>
            <item row="8" column="0">
             <widget class="QTextBrowser" name="clipText_5"/>
            </item>
            <item row="14" column="0">
             <widget class="QTextBrowser" name="clipText_7"/>
            </item>
            <item row="2" column="0">
             <widget class="QTextBrowser" name="clipText_2"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>-1</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Text</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="16" column="0">
             <widget class="QTextBrowser" name="clipText_9"/>
            </item>
            <item row="4" column="0">
             <widget class="QTextBrowser" name="clipText_4"/>
            </item>
            <item row="9" column="0">
             <widget class="QTextBrowser" name="clipText_6"/>
            </item>
            <item row="15" column="0">
             <widget class="QTextBrowser" name="clipText_8"/>
            </item>
           </layout>
          </widget>
          <widget class="QPushButton" name="textCopy_1">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="textCopy_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="textCopy_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>160</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="textCopy_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>220</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="textCopy_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>280</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="textCopy_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>340</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="textCopy_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>400</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="textCopy_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>460</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="textCopy_9">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>520</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="images">
          <attribute name="title">
           <string>Images</string>
          </attribute>
          <widget class="QWidget" name="gridLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>0</y>
             <width>321</width>
             <height>561</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="4" column="0">
             <widget class="QGraphicsView" name="clipImage_4"/>
            </item>
            <item row="0" column="4">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="0">
             <widget class="QGraphicsView" name="clipImage_3"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Dimension</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="imgDim_3">
              <property name="text">
               <string>Dim</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QLabel" name="imgSize">
              <property name="text">
               <string>Size</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="imgDim_4">
              <property name="text">
               <string>Dim</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QGraphicsView" name="clipImage"/>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="imgDim">
              <property name="text">
               <string>Dim</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="5">
             <widget class="QLabel" name="imgSize_5">
              <property name="text">
               <string>Size</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Size</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QLabel" name="imgSize_2">
              <property name="text">
               <string>Size</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QLabel" name="imgSize_3">
              <property name="text">
               <string>Size</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="5">
             <widget class="QLabel" name="imgSize_4">
              <property name="text">
               <string>Size</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="imgDim_5">
              <property name="text">
               <string>Dim</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QGraphicsView" name="clipImage_5"/>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="imgDim_2">
              <property name="text">
               <string>Dim</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <widget class="QGraphicsView" name="clipImage_2"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Image</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QPushButton" name="copy1">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="copy2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>180</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="copy3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>280</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="copy4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>390</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="copy5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>500</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
